packages/fxa-react/configs/tailwind.js

```diff
@@ -312,38 +282,41 @@ module.exports = {
     plugin(function ({ addComponents }) {
       const carets = {
         '.caret-top': {
-          borderLeft: '.75rem solid transparent',
-          borderRight: '.75rem solid transparent',
-          borderBottom: '.75rem solid #fff',
+          borderLeft: '0.75rem solid transparent',
+          borderRight: '0.75rem solid transparent',
+          borderBottom: '0.75rem solid #fff',
         },
         '.caret-top-default': {
-          borderLeft: '.75rem solid transparent',
-          borderRight: '.75rem solid transparent',
-          borderBottom: '.75rem solid #5e5e72',
+          borderLeft: '0.75rem solid transparent',
+          borderRight: '0.75rem solid transparent',
+          borderBottom: '0.75rem solid #5e5e72',
         },
         '.caret-top-error': {
-          borderLeft: '.75rem solid transparent',
-          borderRight: '.75rem solid transparent',
-          borderBottom: '.75rem solid #E22850',
+          borderLeft: '0.75rem solid transparent',
+          borderRight: '0.75rem solid transparent',
+          borderBottom: '0.75rem solid #E22850',
         },
         '.caret-bottom': {
-          borderLeft: '.75rem solid transparent',
-          borderRight: '.75rem solid transparent',
-          borderBottom: '.75rem solid #fff',
+          borderLeft: '0.75rem solid transparent',
+          borderRight: '0.75rem solid transparent',
+          borderBottom: '0.75rem solid #fff',
         },
         '.caret-bottom-default': {
-          borderLeft: '.75rem solid transparent',
-          borderRight: '.75rem solid transparent',
-          borderTop: '.75rem solid #3D3D3D',
+          borderLeft: '0.75rem solid transparent',
+          borderRight: '0.75rem solid transparent',
+          borderTop: '0.75rem solid #3D3D3D',
         },
         '.caret-bottom-error': {
-          borderLeft: '.75rem solid transparent',
-          borderRight: '.75rem solid transparent',
-          borderTop: '.75rem solid #E22850',
+          borderLeft: '0.75rem solid transparent',
+          borderRight: '0.75rem solid transparent',
+          borderTop: '0.75rem solid #E22850',
         },
       };
       addComponents(carets);
     }),
     tailwindcssDir(),
   ],
+  ...(process.env.NODE_ENV === 'development' && {
+    safelist: [{ pattern: /.*/ }],
+  }),
```

Comment:
@LZoog I know this was part of a different ticket (#FXA-6235), but I've added it here since I was already editing the config. I've tested that this allows adding tailwind utilities directly in dev tools.

Reply:
@vpomerleau Do you mind adding a comment above this line saying what this is for? Even something as simple as, "Disables JIT in development" works. This is great btw, glad it works in dev and in Storybook (just verified). 😁